playbooks/roles/logstash/templates/logstash.conf.j2

```diff
@@ -1,43 +1,197 @@
+    grok {
+      match => { "message" => "%{TIMESTAMP_ISO8601:syslog_timestamp} %{LOGLEVEL:log_level} %{POSINT:syslog_pid} \[%{DATA:queue_service}\] %{DATA:filename}\:%{INT:line_number} \- %{GREEDYDATA:syslog_message}" }
+    }
+  } else if ("/edx/app/edxapp/edx-platform/logs/util1/supervisor/xqueue_consumer-stderr.log" in [source] or
+    "/edx/app/edxapp/edx-platform/logs/util1/supervisor/certs-stderr.log" in [source]) {
+    grok {
+      match => { "message" => "%{GREEDYDATA:syslog_message}" }
+    }
+  } else if ("{{ COMMON_LOG_DIR }}/devpi/supervisor/supervisord.log" in [source]) {
+    grok {
+      match => { "message" => "%{GREEDYDATA:syslog_message}" }
+    }
+  } else if ("{{ COMMON_LOG_DIR }}/rabbitmq/edx.cms.core.default.log" in [source]) {
+    grok {
+      match => { "message" => "%{WORD}\=%{QUOTEDSTRING:syslog_timestamp} %{WORD}\=%{QUOTEDSTRING:rabbitmq_vhost} %{WORD}\=%{QUOTEDSTRING:rabbitmq_queue} %{WORD}\=%{INT:rabbitmq_length}" }
+    }
+  } else if ("{{ COMMON_LOG_DIR }}/mongo/mongodb.log" in [source]) {
+    grok {
+      match => { "message" => "%{TIMESTAMP_ISO8601:syslog_timestamp} ?\[?%{DATA:mongo_context}?\]? %{GREEDYDATA:syslog_message}" }
+    } 
+  } else if ("{{ COMMON_LOG_DIR }}/tracking/tracking.log" in [source] or 
+    "/edx/app/edxapp/edx-platform/logs/app1/tracking/tracking.log" in [source]) {
+    grok {
+      match => { "message" => "%{GREEDYDATA:syslog_message}" }
+    }
+  } else if ("{{ logstash_log_dir }}/logstash.log" in [source]) {
+    grok {
+      match => { "message" => "\{\:%{WORD}\=\>\"%{TIMESTAMP_ISO8601:syslog_timestamp}\"\, %{GREEDYDATA:syslog_message}\}" }
+    }
+  } else if ("{{ elasticsearch_log_dir }}/elasticsearch.log" in [source]) {
+    grok {
+      match => { "message" => "(?:\[%{TIMESTAMP_ISO8601:syslog_timestamp}\]\[%{WORD:log_level}? \])%{GREEDYDATA:syslog_message}" }
+    }
+  } else { #matches all rsyslog messages
+    grok {
+      match => { "message" => "%{SYSLOGTIMESTAMP:syslog_timestamp} %{WORD:syslog_hostname} %{DATA:syslog_program}(?:\[%{POSINT:syslog_pid}\])?: %{GREEDYDATA:syslog_message}" }
+      match => { "message" => "%{SYSLOGTIMESTAMP:syslog_timestamp} %{WORD:syslog_hostname} \[%{WORD}\=%{DATA:service_variant}\]\[%{DATA:service_program}\]\[%{WORD}\:%{USERNAME:env}\] %{WORD:log_level} \[%{WORD} ? %{POSINT:syslog_pid}\] \[%{DATA:filename}\:%{INT:line_number}\] \- %{GREEDYDATA:syslog_message}" }
+      tag_on_failure => []
+    }
+  }
+  }
+  syslog_pri { }
+  date {
+    match => [ "syslog_timestamp", "MMM  d HH:mm:ss", "MMM dd HH:mm:ss", "DD MMM YYYY:HH:mm:ss ZZZ", "ISO8601"]
+  }
+  # Try and parse the tracking log json
+  # 142 is syslog facility 17 (local1) and Informational.
+  # This is used to reduce the number of errors in json parsing as
+  # tracking uses that facility and priority by default.
+  if "142" in [syslog_pri] {
+    json {
+      source => "syslog_message"
+      target => "tracking"
+    }
+  }
+#  if ([message] =~ /(?i)warn/) {
```

Comment:
As long as we're properly parsing the severity out of syslog, I think we can go ahead and remove this now.